src/BuiltInTools/HotReloadClient/DefaultHotReloadClient.cs

```diff
@@ -28,6 +28,9 @@ internal sealed class DefaultHotReloadClient(ILogger logger, ILogger agentLogger
         private NamedPipeServerStream? _pipe;
         private bool _managedCodeUpdateFailedOrCancelled;
 
+        // The status of the last update response.
+        private TaskCompletionSource<bool> _updateStatusSource = new();
```

Comment:
Should this be disposed?

Reply:
TaskCompletionSource is not disposable.

Reply:
Derp! Twas thinking of `CancellationTokenSource`. :smile: